test/test_sparql/test_functions.py

```diff
@@ -167,7 +167,7 @@ def test_function(expression: str, expected_result: Identifier) -> None:
     if isinstance(expected_result, type):
         assert isinstance(actual_result, expected_result)
     else:
-        assert expected_result == actual_result
+        assert actual_result == expected_result
```

Comment:
This change was made to help the test suite output.
PyTest always assumes when asserting equality that LHS is actual, and RHS is expected, but this test harness compared them the other way around, so PyTest output was backwards that made debugging more difficult.